dist/index.css

```diff
@@ -1,3 +1,4 @@
+@import url("https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.4.1/components/icon.min.css");
```

Comment:
we should not use this css for the icon, ask Vencent

Reply:
done, wasn't using it, just forget to rebuild the library after I use Vincent's Icon